FrontEnd/src/lib/schemas/loginSchema.ts

```diff
@@ -0,0 +1,12 @@
+import { z } from "zod";
+
+export const loginSchema = z.object({
+  email: z
+    .string()
+    .min(1, "Correo o usuario obligatorio"),
+  password: z
+    .string()
+    .min(1, "Contraseña obligatoria")
```

Comment:
```suggestion
    .min(6, "mínimo 6 caracteres")
```
añade mensaje de error para contraseñas menores a 6 caracteres